internal/armadactl/queue.go

```diff
@@ -4,22 +4,42 @@ import (
 	"fmt"
 	"sort"
 
-	"github.com/G-Research/armada/pkg/api"
+	"github.com/G-Research/armada/pkg/client"
+	"github.com/G-Research/armada/pkg/client/queue"
+	"github.com/G-Research/armada/pkg/client/util"
 )
 
 // CreateQueue calls app.QueueAPI.Create with the provided parameters.
-func (a *App) CreateQueue(name string, priorityFactor float64, owners []string, groups []string, resourceLimits map[string]float64) error {
-	queue := api.Queue{
-		Name:           name,
-		PriorityFactor: priorityFactor,
-		UserOwners:     owners,
-		GroupOwners:    groups,
-		ResourceLimits: resourceLimits,
-	}
+func (a *App) CreateQueue(queue queue.Queue) error {
 	if err := a.Params.QueueAPI.Create(queue); err != nil {
-		return fmt.Errorf("[armadactl.CreateQueue] error creating queue %s: %s", name, err)
+		return fmt.Errorf("[armadactl.CreateQueue] error creating queue %s: %s", queue.Name, err)
+	}
+	fmt.Fprintf(a.Out, "Created queue %s\n", queue.Name)
+	return nil
+}
+
+func (a *App) CreateResource(fileName string, dryRun bool) error {
+	var resource client.Resource
+	if err := util.BindJsonOrYaml(fileName, &resource); err != nil {
+		return fmt.Errorf("file %s error :%s", fileName, err)
```

Comment:
```suggestion
		return fmt.Errorf("file %s error: %s", fileName, err)
```

Reply:
Will this produce an error message that is nicely formatted and easily understood by the user? If not, I think we should change it. Same comment for the other error messages in this function.